lib/zuora/objects/invoice.rb

```diff
@@ -15,20 +15,20 @@ class Invoice < Base
     validates_numericality_of :amount
     validates_numericality_of :balance, :allow_nil => true
     validates_length_of       :comments, :maximum => 255
-    validates_datetime_of     :due_date
-    validates_datetime_of     :invoice_date
+    # validates_datetime_of     :due_date
```

Comment:
I'm not sure why these validations are commented. Do we need to compare `zuora.a.78.0.wsdl` with `zuora.a.38.0.wsdl` and find out if these fields are removed in latest file.

Reply:
Reverted in this commit [841f53b](https://github.com/Sage/zuora/pull/18/commits/841f53b823eaa50e77a234490d96a9cee4bdc218)